reverse.js

```diff
@@ -0,0 +1,12 @@
+let arr = [1, 2, 3, 6, 4];
+
+function reverse (arr){
+  let newArr = [];
+  for (let i = arr.length - 1; i >= 0; i--){
+    newArr.push(arr[i]);
+  }
+  arr = newArr
+  return arr;
```

Comment:
This is a better solution, but try to do it in a better way.

1. without a new array
2. iterate until the half of the array